Source/DafnyCore/Util.cs

```diff
@@ -195,6 +196,9 @@ public static class Util {
       UnescapedCharacters(s, isVerbatimString).Iter(ch => sb.Append(ch));
       return sb.ToString();
     }
+
+    public static readonly Regex Utf16Escape = new Regex(@"(?<!\\)\\u([0-9a-fA-F]{4})");
```

Comment:
This doesn't look right: how about `\\\uAAAA`?  It won't match, right?

Reply:
Note that both of those other bogus regex patterns will be removed entirely in the fix to https://github.com/dafny-lang/dafny/issues/2928, since they are trying to do entirely the wrong thing anyway.